packages/yew/Cargo.toml

```diff
@@ -22,7 +22,7 @@ gloo = "0.8"
 indexmap = { version = "1", features = ["std"] }
 js-sys = "0.3"
 slab = "0.4"
-wasm-bindgen = "0.2"
+wasm-bindgen = { version = "0.2", features = ["enable-interning"] }
```

Comment:
I don't think we should enable this feature on the dependency here ourselves. It impacts all usages of [wasm_bindgen::intern](https://rustwasm.github.io/wasm-bindgen/api/wasm_bindgen/fn.intern.html#), even ones outside of yew. This feature is, to my understanding, something the end user should control as part of the executable, not dictated by libraries. If this feature is not enabled, the above method is a no-op, it will still work.

Instead, we should document somewhere on the [optimizations page](https://yew.rs/docs/next/advanced-topics/optimizations) how to enable string interning and that we intern keys and event types if enabled.

Reply:
Yes I think that's the correct decision. I can revert this and — what do you think, add an `enable-interning` flag to Yew? Something by another name?

Reply:
There is nothing that needs to be done in yew itself, I think. Thanks to [feature unification](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification), the end user can make sure that the feature is enabled, if they want to use it. This will then affect also the uses of interning inside yew. So basically the docs could say

> Yew uses [string interning](https://rustwasm.github.io/wasm-bindgen/api/wasm_bindgen/fn.intern.html#) on attribute names and event types. This can speed up element creation and updates to the DOM. To opt-in to this behaviour, enabled the `enable-interning` feature on wasm-bindgen in the `Cargo.toml` of your app.
>
> ```
>  wasm-bindgen = { version = "0.2", features = ["enable-interning"] }
> ```

Reply:
The users may not want to add a direct dependency on wasm-bindgen so we can add a feature flag that enables the feature for wasm-bindgen. That shouldn't interfere with feature unification and gives the option to enable interning on yew

Reply:
An optional default-off feature that just enables the `wasm-bindgen/enable-interning` feature sounds reasonable to me, too.